code/modules/food_and_drinks/restaurant/generic_venues.dm

```diff
@@ -83,7 +83,7 @@
 	)
 
 /datum/venue/bar/get_food_appearance(order)
-	var/datum/reagent/reagent_to_order = order
+	var/datum/reagent/reagent_to_order = new order
```

Comment:
do var/datum/glass_style/draw_as = GLOB.glass_style_singletons[potential_container][initial(reagent_to_order.type)] 
instead in the runtiming line

Reply:
@Fikou I tried this in dfa7d8c3641a2c03323607f950d67bc7b86c5766, and it doesn't compile since `type` is static and unchangeable, so BYOND doesn't accept it for whatever reason. It has to be instantiated this way (unless there's something much more obscure that I can find).